hack/machineset.sh

```diff
@@ -78,7 +78,7 @@ get_aws_ms() {
   local provider=$4
 
   # get the AMI id for the Windows VM
-  ami_date="2020.09.09"
+  ami_date="2020.10.14"
```

Comment:
Please add a check after `ami_id` is populated to throw an error if `ami_id` is empty,

Reply:
updated.